### ISSUE TITLE
TypeError `Object.values` is not a function as `useQuery`

### ISSUE THREAD

**Issue by shabab477** [apollographql/apollo-client#9376]
**Intended outcome:**

I have this line in the code:  

`const { loading, error, data } = useQuery(GET_DIAGNOSIS);`

Which throws an error **TypeError `Object.values` is not a function**.

**How to reproduce the issue:**
This issue is taking place at:

Chrome Mobile, Version: 50.0.2661
Android Version: 4.3

**Versions**

System:
    OS: Linux 5.11 Ubuntu 20.04.3 LTS (Focal Fossa)
  Binaries:
    Node: 14.16.1 - /usr/local/bin/node
    Yarn: 1.22.15 - /usr/local/bin/yarn
    npm: 6.14.12 - /usr/local/bin/npm
  Browsers:
    Chrome: 97.0.4692.99
    Firefox: 96.0
  npmPackages:
    @apollo/client: ^3.5.7 => 3.5.7

**Comment by brainkim** [comment]
Hi! You might need a shim for `Object.values()`. According to [caniuse](https://caniuse.com/?search=Object.values), it’s not supported in the Chrome version that you are using.

**Comment by shabab477** [comment]
@brainkim yes, I am using a [polyfill bundle](https://polyfill.io/v3/url-builder/) to mimic `Object.values`. Was wondering whether it's a good idea that Apollo uses the shim internally though. Happy to close the issue either ways.

**Comment by phryneas** [comment]
Hi everyone, I'm doing some housekeeping on old issue.

I am going this issue as `Object.values` it widely supported as of today - generally, please see our [versioning policy](https://github.com/apollographql/apollo-client/blob/main/VERSIONING_POLICY.md) that explains how we determine which browser features we support, and which might need to be shimmed. 
We generally try to avoid shipping polyfills as those add bundle size to everyone, even though just a very small fraction of our user base benefits from them. 

Have a great day everyone!
